Makefile

```diff
@@ -40,7 +40,7 @@ $(call add-crd-gen,ocsv1,./pkg/apis/ocs/v1,./deploy/crds,./deploy/crds)
 	shellcheck-test \
 	gofmt \
 	golint \
-	govet \
+	gitgovet \
```

Comment:
Isn't the target still named `govet`? At least it is in this version of the file.

Reply:
Yes, you are right, I guess I just wrote `git` while the focus was on the editor.
I will revert and push

Reply:
done